classes/image_types_fsl.bbclass

```diff
@@ -9,19 +9,19 @@ UBOOT_SUFFIX ?= "bin"
 MXSBOOT_NAND_ARGS ?= ""
 
 # U-Boot mxsboot generation for uSD
-do_image_uboot_mxsboot_sdcard[depends] += "u-boot-mxsboot-native:do_populate_sysroot \
+do_image_uboot:mxsboot_sdcard[depends] += "u-boot-mxsboot-native:do_populate_sysroot \
                                            u-boot:do_deploy"
-IMAGE_CMD_uboot-mxsboot-sdcard() {
+IMAGE_CMD:uboot-mxsboot-sdcard() {
     mxsboot sd ${DEPLOY_DIR_IMAGE}/u-boot-${MACHINE}.${UBOOT_SUFFIX} \
                ${DEPLOY_DIR_IMAGE}/${IMAGE_NAME}.rootfs.uboot-mxsboot-sdcard
     ln -sf ${IMAGE_NAME}.rootfs.uboot-mxsboot-sdcard \
            ${DEPLOY_DIR_IMAGE}/${IMAGE_LINK_NAME}.rootfs.uboot-mxsboot-sdcard
 }
 
 # U-Boot mxsboot generation for NAND
-do_image_uboot_mxsboot_nand[depends] += "u-boot-mxsboot-native:do_populate_sysroot \
+do_image_uboot:mxsboot_nand[depends] += "u-boot-mxsboot-native:do_populate_sysroot \
```

Comment:
Seems broken. @twoerner see if using `do_image:uboot_mxsboot_nand` here and other places might help.

Reply:
ah yes confused script, I pushed a fix in branch